budget_management_project/expense/serializers.py

```diff
@@ -0,0 +1,37 @@
+from django.utils import timezone
+from rest_framework import serializers
+
+from expense.models import Expense
+
+class ExpenseSerializer(serializers.ModelSerializer):
+    category_name = serializers.CharField(source='category.name', read_only=True)
+    formatted_money = serializers.SerializerMethodField()
+    user = serializers.HiddenField(default=serializers.CurrentUserDefault())
+
+    class Meta:
+        model = Expense
+        fields = [
+            'id',
+            'user',
+            'category',
+            'category_name',
+            'expense_money',
+            'formatted_money',
+            'expense_date',
+            'memo',
+            'created_at'
+        ]
+        read_only_fields = ['id', 'created_at']
+
+    def validate_expense_money(self, value):
+        if value <= 0:
+            raise serializers.ValidationError("지출 금액은 0보다 커야 합니다.")
+        return value
```

Comment:
이 부분 필드단에서 최소값 설정으로 간단하게 할 수 있습니다
```suggestion
    expense_money = serializers.IntegerField(min_value=1)
```